src/main/groovy/com/github/dispader/manifesto/ManifestoPlugin.groovy

```diff
@@ -12,7 +11,7 @@ class ManifestoPlugin implements Plugin<Project> {
         project.extensions.create('manifesto', ManifestoPluginExtension, project)
 
         project.plugins.whenPluginAdded { plugin ->
-            project.tasks.findAll { ( it instanceof Jar || it instanceof War ) }.each {
+            project.tasks.withType(Jar).each {
```

Comment:
Hey @cdinger — I don't think the change here does what you think, but it's a really awesome change nonetheless.

This code isn't checking that the `war` plugin or `jar` plugin is loaded, but modifying configurations for the tasks that archive things up.

However — [the `War` task actually does extend the `Jar` task](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/bundling/War.html) — and this change not only means we don't pick and choose specific archives, but makes the plugin capable of configuring manifests for anything inheriting from `Jar`.

💯